platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/EventRecoveryWorkflow.java

```diff
@@ -124,30 +126,36 @@ public static void recoverState(
 
         logger.info(STARTUP.getMarker(), "Loading state from {}", signedStateFile);
 
-        final SignedState initialState =
-                SignedStateFileReader.readStateFile(signedStateFile).signedState();
+        final PlatformContext platformContext = new DefaultPlatformContext(
```

Comment:
Because this class needs to read state files from disk, and to avoid using static getter methods we now pass a context into the static methods that read the state file.

Reply:
This class is only invoked from a PCLI command, but I agree the platform context should be created there and passed in. Captured in this ticket https://github.com/hashgraph/hedera-services/issues/6300